jdk11-tests/src/main/java/module-info.java

```diff
@@ -27,8 +27,9 @@
     // and datatypes
     requires com.fasterxml.jackson.datatype.joda;
     requires com.fasterxml.jackson.datatype.guava;
-    requires transitive joda.time;
+    requires static joda.time;
```

Comment:
I like static for these sort of things because it allows the implementer to change their module name (when switching from automatic to named e.g.) and not affecting the libraries that are dependent on it.

So I generally go by - If we expect them to change it later, mark it static to allow the user to change the version

Reply:
Ok. So here it's not a big deal, but to me things get bit tricky in other cases -- would you consider using `static` by, say, Jackson Joda datatype module? While I understand benefit of name change (until Module names stabilize more), doesn't it require more effort from user's part, to add additional dependency? (since dependency is not truly optional; module would fail to load if not specified).

Reply:
It does force the consumer to add the ion/joda/avro/guice/jaxb libraries doing it this way yes - 

But with the module names changing and jpackage coming in jdk 13, this really does do the trick

Reply:
I mean the maven/jar libraries - if the module is found on the classpath, then the package works. 

This also only affects JDK 9 ^ apps that are specifically running in modular (with the -m java directive), backwards compatibility kept.

Have I sold you over yet? *holds thumbs*

Reply:
Well since this is a test repo I guess it does not greatly matter either way. I'll probably understand this better once I get past JDK 8 myself.